airsonic-main/src/test/java/org/airsonic/player/dao/UserDaoTestCase.java

```diff
@@ -120,7 +120,7 @@ public void testGetUserByName() {
         assertUserEquals(user, newUser);
 
         assertNull("Error in getUserByName().", userDao.getUserByName("sindre2", true));
-        assertNull("Error in getUserByName().", userDao.getUserByName("sindre ", true));
+        assertNotNull("Error in getUserByName().", userDao.getUserByName("sindre ", true));
```

Comment:
vaguely remember fixing this test... I'm not sure why this is behaving differently

Reply:
See ae4d49f59a0b7f10d11648699391888993ad8e37.

Reply:
Yep, and https://github.com/airsonic/airsonic/pull/847/files#diff-979b441261c64e3f4523f0e4fe90711dR118